src/lib/server/session.cpp

```diff
@@ -25,9 +25,7 @@ void Session::run() {
   while (!_terminate_session) {
     try {
       _handle_request();
-    } catch (const ClientDisconnectException&) {
-      return;
-    } catch (const std::exception& e) {
+    } catch (const ClientDisconnectException&) { return; } catch (const std::exception& e) {
```

Comment:
Did you intentionally reformat this? It may be debatable whether this is more readable.

Reply:
This is a known issue with clang format. :-/

Reply:
Would be nice to revert that at some later point in time.